app/scripts/directives/resources.js

```diff
@@ -12,7 +81,27 @@ angular.module('openshiftConsole')
         // Optional URL for setting health checks on the resource when missing.
         addHealthCheckUrl: '@?'
       },
-      templateUrl: 'views/_pod-template.html'
+      templateUrl: 'views/_pod-template.html',
+      link: function(scope) {
+        scope.containers = _.get(scope.podTemplate, 'spec.containers', []);
+        scope.initContainers = _.get(scope.podTemplate, 'spec.initContainers', []);
```

Comment:
If we do this assignment here, it won't update when the pod status changes. Easier just to access `podTemplate.spec.containers` or `podTemplate.spec.initContainers` directly in the view.